src/components/Navigation/NavBar.js

```diff
@@ -16,34 +16,39 @@ import FlagGroup from './FlagGroup';
 export const NavBar = React.createClass({
   render() {
     const params = this.props.params;
-    const courseLink = params.course ? <NavLink to={`/${params.course}`}>{params.course}</NavLink> : null;
+    const courseLink = params.course ?
+      <NavLink isStudentMode={this.props.isStudentMode} to={`/${params.course}`}>
+        {params.course}
+      </NavLink> : null;
     const lessonLink = params.course && params.lesson && params.file ?
-      <NavLink to={`/${params.course}/${params.lesson}/${params.file}`}>{params.file}</NavLink> : null;
+      <NavLink isStudentMode={this.props.isStudentMode} to={`/${params.course}/${params.lesson}/${params.file}`}>
+        {params.file}
+      </NavLink> : null;
 
     return (
       <Grid fluid={true}>
         <Row>
-          <FlagGroup />
+          <FlagGroup/>
         </Row>
         <Row>
-          <Navbar fluid={true}>
+          <Navbar className={this.props.isStudentMode ? null : 'navbar-teacher'} fluid={true}>
             <Navbar.Header>
               <Navbar.Brand>
-                <NavLink to="/" onlyActiveOnIndex>
+                <NavLink isStudentMode={this.props.isStudentMode} to="/" onlyActiveOnIndex>
                   <Glyphicon glyph="home"/>
                 </NavLink>
                 {courseLink ? <span> / {courseLink}</span> : null}
                 {lessonLink ? <span className="hidden-xs"> / {lessonLink}</span> : null}
               </Navbar.Brand>
-              <Navbar.Toggle />
+              <Navbar.Toggle className={this.props.isStudentMode ? null : 'toggle-teacher'}/>
             </Navbar.Header>
             <Navbar.Collapse>
               <Navbar.Form pullRight>
                 <FormGroup>
                   <FormControl type="text" placeholder="Søk"/>
                 </FormGroup>
                 {' '}
-                {this.props.isStudentMode ?
+                {!this.props.isStudentMode ?
```

Comment:
Gjøre øvrige endringer av toggleknappen i navbar i egen PR for å unngå mange konflikter i branch?